openbb_platform/core/openbb_core/provider/standard_models/share_price.py

```diff
@@ -0,0 +1,40 @@
+"""Share Prices Standard Model."""
+
+from datetime import date as dateType
+from typing import Optional
+
+from pydantic import Field
+
+from openbb_core.provider.abstract.data import Data
+from openbb_core.provider.abstract.query_params import QueryParams
+from openbb_core.provider.utils.descriptions import (
+    DATA_DESCRIPTIONS,
+    QUERY_DESCRIPTIONS,
+)
+
+
+class SharePriceQueryParams(QueryParams):
+    """Share Price Query."""
+
+    start_date: Optional[dateType] = Field(
+        default=None, description=QUERY_DESCRIPTIONS.get("start_date")
+    )
+    end_date: Optional[dateType] = Field(
+        default=None, description=QUERY_DESCRIPTIONS.get("end_date")
+    )
+
+
+class SharePriceData(Data):
+    """Share Price Data."""
+
+    date: Optional[dateType] = Field(
+        default=None, description=DATA_DESCRIPTIONS.get("date")
+    )
+    value: Optional[float] = Field(
+        default=None,
+        description="Interest rate (given as a whole number, i.e 10=10%)",
```

Comment:
Description here is from something else.  Is the field going to represent a price? 

Maybe there's a better name for this standard model? "GlobalSharePriceIndex" or something more intuitive than "SharePriceData" which could easily be construed for data related to TSLA or whatever else.

Reply:
looks llike its not a requirement anymore, so may yank it